### ISSUE TITLE
Define Charka UI theme

### ISSUE THREAD

**Issue by Myrfion** [Seneca-CDOT/starchart#34]
Since we are using Charka UI I think we will have to define a theme for it. One of the first things that we could put there could be the red colour as a primary colour as it was mentioned in wiki:
```
Use colorScheme='red' for ChakraUI components to match Seneca's red colour theme, see 
https://chakra-ui.com/docs/components/button/usage#usage
```
Charka also provides functionality to override styles of its own components like `Button` in the theme and in my opinion we should use theming as much as possible in order to keep all or almost all styles in one place. 

Read more on Charka UI theming here: https://chakra-ui.com/docs/styled-system/theme

**Comment by humphd** [comment]
@Myrfion when you make a PR, you need to use "Fixes #34" in the description so it links the PR with the Issue, and auto-closes it.  I think #100 fixed this, correct?  If so, please close this.

**Comment by Myrfion** [comment]
Fixed in #100
